Disfunction/Assets/Game Scripts/Gun Behaviour/GunMechanics.cs

```diff
@@ -31,6 +32,9 @@ public class GunMechanics : MonoBehaviour
     public float kickBackPower;
     public float fireRate = 0.1f;
     public float sensitivity = 1;
+    public float damage = 10.0f;

```

Comment:
Change these variable names to impactWithinMeduimRange / impactWithinFarRange (Something like that)